pages/en/learn/getting-started/nodejs-with-typescript.md

```diff
@@ -148,20 +148,26 @@ Then you can run your TypeScript code like this:
 npx ts-node example.ts
 ```
 
-### Running TypeScript Code with a Node.js Loader
+### Running TypeScript Code with `tsx`
 
-Since Node.js v19.0.0, you can use a [custom loader](https://nodejs.org/api/cli.html#--importmodule). Download a loader such as `ts-node` or `tsx` or `nodejs-loaders`
```

Comment:
I'm not fan of removing "since of nodejs v19.0.0". 
Because this tells you which version the example is running on. But you could simply add the words “on the current LTS you can do ....”.

Reply:
Isn’t it assumed that the examples in the Learn pages are all for the current version of Node, or at least the current supported versions?

Reply:
Yes, it's true. Maybe we should mention it somewhere, because for the moment it's not written anywhere.

Reply:
Sure, but I assume that should be a separate PR. Is there anything else about this or can I consider this resolved?